doc/meetings/2024_01_15.md

```diff
@@ -0,0 +1,73 @@
+# Meeting 2023-01-15
+
+## Attendees
+
+- Markus
+- Yvonne
+- Daniel (Protocol done)
+- Jannis (Protocol done)
+- Christoph (Protocol done)
+- Moritz
+
+## Buddies
+
+- Moritz & Christoph
+- Daniel & Jannis
+
+## Agenda
+
+- 09:00 Kennenlernen+Checkin
+- Protocol: Moritz
+- shadings tests
+- release
+- Samuel
+- please do "Tasks for Everyone" and individual tasks
+  - get PRs and UCs done
+  - please meet deadlines, so that reviews etc. can be done
+- [API documentation](../backend/03api_documentation.md)
+- median
+- [sprint plan](https://project.permaplant.net)
+
+## Tasks for Everyone
+
+To be done until **Friday 19.01.2024**:
+
+1. buddy talk topic: improvement of (API) documentation
+2. do/update [sprint plan](https://project.permaplant.net)
+3. approve meeting PR [#1157](https://pull.permaplant.net/1157/files),
+   check for [requested reviews](https://pulls.permaplant.net/?q=is%3Aopen+user-review-requested%3A%40me)
+
+If possible/needed do:
+
+- create/update issues/tests (behavior driven) as needed for future tasks
+- get current issues, PRs and use cases (UC) [done](../usecases/README.md)
+- do reviews of PRs in main repo and submissions
+- submit a (short) text in submissions
+
+## Individual Tasks
+
+To be done until **Friday 19.01.2024**:
+
+- Daniel: endpoint requirements, notes
+- Jannis:notes, API refactoring,
+  doc/decisions/backend_orm_crate.md,
+  (timeline)
+- Christoph: landingpage, spread&height, review heatmap+db docu
+- Moritz: group instead of layers, create shade layer UC issues (8-edges, size selectable, overlap)
+- Yvonne: Shading Testen
+- Markus: Release
+
+## Meeting Notes
+
+Raw SQL queries should be preferred over typed queries for increased developer ergonomics.
```

Comment:
I don't understand this point. Will this be a guideline or decision?

Reply:
@horenso will look into if Diesel raw SQL works nicely. If the problems with Diesel persist, we reopen the decision and look into alternatives again (also the one you mentioned).

If Diesel raw SQL works good enough, it is only one guideline to add to our documentation and most likely not the sentence here in the meeting notes (raw SQL is rather Plan B).